Project2015To2017/Transforms/FileTransformation.cs

```diff
@@ -127,15 +133,30 @@ private static bool KeepFileInclusion(XElement x)
 				}
 			}
 
-			var filesInFolder = projectFolder.EnumerateFiles(wildcard, SearchOption.AllDirectories).Select(x => x.FullName.ToUpper()).ToArray();
+			var filesInFolder = projectFolder.EnumerateFiles(wildcard, SearchOption.AllDirectories).Select(x => x.FullName).ToArray();
```

Comment:
Why did you remove this ToUpper? That is to prevent an issue where the names of the files doesn't match what's in the csproj file. Windows doesn't care but Linux might.